content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,381 @@
+{{< /note >}}
+
+Let's perform a rollout for the Deployment that will cause all the Pods to restart.
+```shell
+kubectl rollout restart deployment configmap-env-var
+```
+
+Check the deployment.
+```shell
+kubectl get deployment configmap-env-var
+```
+
+You should see an output similar to:
+```
+NAME                READY   UP-TO-DATE   AVAILABLE   AGE
+configmap-env-var   3/3     3            3           12m
+```
+
+Check the Pods.
+```shell
+kubectl get pods --selector=app=configmap-env-var
+```
+
+The older Pods will terminate and new ones will be created. You should finally see an output similar to:
+```
+NAME                                 READY   STATUS        RESTARTS   AGE
+configmap-env-var-6d94d89bf5-2ph2l   1/1     Running       0          13s
+configmap-env-var-6d94d89bf5-74twx   1/1     Running       0          8s
+configmap-env-var-6d94d89bf5-d5vx8   1/1     Running       0          11s
+```
+
+View the logs of the Deployment.
+```shell
+kubectl logs deployment/configmap-env-var
+```
+
+You should see an output similar to the below:
+```
+Found 3 pods, using pod/configmap-env-var-6d9ff89fb6-bzcf6
+Thu Jan  4 16:30:35 UTC 2024 The basket is full of mangoes
+Thu Jan  4 16:30:45 UTC 2024 The basket is full of mangoes
+Thu Jan  4 16:30:55 UTC 2024 The basket is full of mangoes
+```
+
+This demonstrates the scenario of updating environment variables in a Pod that are derived from a ConfigMap. Changes to the ConfigMap values are applied to the Pod during the subsequent rollout.  
+
+## Update configuration via a ConfigMap in a multi-container Pod
+
+Use the `kubectl create configmap` command to create a ConfigMap from [literal values](/docs/tasks/configure-pod-container/configure-pod-configmap/#create-configmaps-from-literal-values):
+
+```shell
+kubectl create configmap colour --from-literal=colour=red
+```
+
+Below is an example of a Deployment having Pods with two containers. The two containers share an ```emptyDir``` Volume that they use to communicate. The first container runs a ```nginx``` server. The mount path for the shared Volume is ```/usr/share/nginx/html```. The second container is based on the ```alpine``` image, and has a mount path of ```/pod-data```. The ```alpine``` container writes a file in HTML which has its content based on a ConfigMap. The ```nginx``` container renders the HTML via HTTP.
+{{% code_sample file="deployments/deployment-with-configmap-two-containers.yaml" %}}
+
+Create the deployment.
+```shell
+kubectl apply -f https://k8s.io/examples/deployments/deployment-with-configmap-two-containers.yaml
+```
+
+Check the deployment.
+```shell
+kubectl get deployment configmap-two-containers
+```
+
+You should see an output similar to:
+```shell
+NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
+configmap-two-containers   3/3     3            3           12s
+
+```
+
+Check the pods.
```

Comment:
```suggestion
Check the pods:
```

Reply:
Ack, will be handled in the next commit.